ecommerce-cart/src/test/kotlin/com/kotato/context/ecommerce/infrastructure/CqrsQueryBusTest.kt

```diff
@@ -0,0 +1,46 @@
+package com.kotato.context.ecommerce.infrastructure
+
+import com.kotato.EcommerceApplication
+import com.kotato.cqrs.domain.query.Query
+import com.kotato.cqrs.domain.query.QueryBus
+import com.kotato.cqrs.domain.query.QueryHandler
+import com.kotato.cqrs.domain.query.ask
+import org.axonframework.queryhandling.NoHandlerForQueryException
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import javax.inject.Inject
+import javax.inject.Named
+import kotlin.test.assertFailsWith
+import kotlin.test.assertTrue
+
+@SpringBootTest(classes = arrayOf(EcommerceApplication::class))
```

Comment:
inherit this test class from ContextStarterTest  instead of using custom context